pandas/tests/io/parser/test_textreader.py

```diff
@@ -341,6 +341,16 @@ def test_empty_csv_input(self):
         df = read_csv(StringIO(), chunksize=20, header=None, names=["a", "b", "c"])
         assert isinstance(df, TextFileReader)
 
+    def test_blank_lines_between_header_and_data_rows(self):
+        ref = DataFrame(
+            [[np.nan, np.nan], [np.nan, np.nan], [1, 2], [np.nan, np.nan], [3, 4]],
+            columns=list("ab"),
+        )
+        csv = "\nheader\n\na,b\n\n\n1,2\n\n3,4"
+        for nrows in range(1, 6):
```

Comment:
Instead of doing this can you just parametrize on the values that matter?

Reply:
Can you elaborate which values you would like to see parameterized?
I basically took the example in the ticket to reproduce the issue and checked that for all sensible values for `nrows` I get the expected outcome.
The reason for this is that this covers more cases that could cause errors like the one in the original ticket due to various edge cases.

Reply:
@roberthdevries : @WillAyd is asking that you use `pytest.mark.parameterize` over the `nrows` parameter.

Reply:
fixed